src/main/java/teammates/common/util/ActivityLogEntry.java

```diff
@@ -386,10 +386,7 @@ public String getActionInfo(){
         return "<a href=\""+getUrlToShow()+"\" class=\""+style+"\" target=\"_blank\">"+servletName+"</a>";
     }
     
-    public String getMessageInfo(){
-        
-        Sanitizer.sanitizeForHtml(message);
-        
+    public String getMessageInfo(){        
```

Comment:
That's a good point! Resolved in 4373d532e587b625c78e80bf80ceee92c7505070. Thanks for pointing out :smiley: